projects/storefrontstyles/scss/_versioning.scss

```diff
@@ -39,7 +39,7 @@ $styleVersion: $_majorVersion !default;
 // For Spartacus development or demo's, the usage of latest styles is recommended.
 // To avoid frequent changes of the $minorVersion in the application, we offer a
 // flag that can be used to keep up with the latest version at any time.
-$useLatestStyles: false !default;
+$useLatestStyles: true !default;
```

Comment:
[MAJOR REMARK - breaking change]
IMHO we shouldn't change this flag to true. It opts-in all customers for using the latest styles, regardless the _feature level_ they configured in their SCSS!
But then please make sure to bump the SCSS feature level used _only in our storefrontapp_ (in our repo) to 6.5